airbyte-integrations/connectors/source-sentry/source_sentry/streams.py

```diff
@@ -1,265 +0,0 @@
-        state_value = self.validate_state_value(stream_state.get(self.cursor_field, self.start_date) if stream_state else self.start_date)
-        return pendulum.parse(state_value)
-
-    def filter_by_state(self, stream_state: Mapping[str, Any] = None, record: Mapping[str, Any] = None) -> Iterable:
-        """
-        Endpoint does not provide query filtering params, but they provide us
-        cursor field in most cases, so we used that as incremental filtering
-        during the parsing.
-        """
-        if pendulum.parse(record[self.cursor_field]) > self.get_state_value(stream_state):
-            # Persist state.
-            # There is a bug in state setter: because of self._cursor_value is not defined it raises Attribute error
-            # which is ignored in airbyte_cdk/sources/abstract_source.py:320 and we have an empty state in return
-            # See: https://github.com/airbytehq/oncall/issues/1317
-            self.state = record
-            yield record
-
-    def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[MutableMapping]:
-        json_response = response.json() or []
-
-        for record in json_response:
-            yield from self.filter_by_state(stream_state=stream_state, record=record)
-
-    @property
-    def state(self) -> Mapping[str, Any]:
-        return {self.cursor_field: self._cursor_value}
-
-    @state.setter
-    def state(self, value: Mapping[str, Any]):
-        """
-        Define state as a max between given value and current state
-        """
-        if not self._cursor_value:
-            self._cursor_value = value.get(self.cursor_field)
-        else:
-            current_value = value.get(self.cursor_field) or self.start_date
-            current_state = str(self.get_state_value(self.state))
-            self._cursor_value = max(current_value, current_state)
-
-
-class Events(SentryIncremental):
-    """
-    Docs: https://docs.sentry.io/api/events/list-a-projects-error-events/
-    """
-
-    primary_key = "id"
-    cursor_field = "dateCreated"
-
-    def __init__(self, organization: str, project: str, **kwargs):
-        super().__init__(**kwargs)
-        self._organization = organization
-        self._project = project
-
-    def path(
-        self,
-        stream_state: Optional[Mapping[str, Any]] = None,
-        stream_slice: Optional[Mapping[str, Any]] = None,
-        next_page_token: Optional[Mapping[str, Any]] = None,
-    ) -> str:
-        return f"projects/{self._organization}/{self._project}/events/"
-
-    def request_params(
-        self,
-        stream_state: Mapping[str, Any],
-        stream_slice: Optional[Mapping[str, Any]] = None,
-        next_page_token: Optional[Mapping[str, Any]] = None,
-    ) -> MutableMapping[str, Any]:
-        params = super().request_params(stream_state, stream_slice, next_page_token)
-        params.update({"full": "true"})
```

Comment:
Please check migration of parameters, (and for the others stream also)

Reply:
thanks. updated params in low code